apps/browser/qml/pages/components/TabView.qml

```diff
@@ -203,10 +200,84 @@ Item {
 
         ]
     }
+
     TabsToolBar {
         id: tabsToolBar
         anchors.bottom: parent.bottom
         onBack: pageStack.pop()
-        onEnterNewTabUrl: root.enterNewTabUrl()
+        onEnterNewTabUrl: tabView.enterNewTabUrl()
+        onOpenMenu: popupMenu.active = true
+    }
+
+    PopUpMenu {
+        id: popupMenu
+
+        signal closeAllTabs
+
+        anchors.fill: parent
+        active: false
+
+        menuItem: Component {
+            Item {
+                id: menuItem_
+                readonly property int iconWidth: Theme.iconSizeMedium + Theme.paddingLarge
+                readonly property int verticalPadding: 3 * Theme.paddingSmall
+                height: content.height + verticalPadding * 2
```

Comment:
Because the `PopupMenu` already has a margin at the bottom set to `Theme.paddingMedium`, you end up with a larger margin at the bottom than at the top. Personally I think it looks better with the same margin at the top and bottom; perhaps it could be balanced unless this is intentional?

Reply:
done

Reply:
There still seems to be something a bit strange going on here. Maybe due to the introduction of the handle the top item is now cut off along the top, See the screenshot, with the top margin highlighted red and comparison with the main browser toolbar menu. I think the highlight bar (when pressing an entry) should be the same height for all entries in the menu.

![top-margin](https://user-images.githubusercontent.com/1446122/123792364-3306df00-d8e9-11eb-8840-bd036317aca2.png)

Reply:
I completely agree, done

Reply:
Great; thank you, it looks well balanced now.